.gitignore

```diff
@@ -1,3 +1,4 @@
 node_modules/
 test/**/package-lock.json
 test/**/yarn.lock
+test/**/dist_electron
```

Comment:
Isn't this already covered by `./test/vue-app/.gitignore`?

Reply:
yes and no - for git, yes, for the 'pretty' command, no, see package.json's 'pretty' command -- it uses the paths from gitignore to figure out what to prettify

Reply:
Good point 👍